packages/framework/src/Twig/PriceExtension.php

```diff
@@ -67,22 +68,35 @@ class PriceExtension extends Twig_Extension
      * @param \Shopsys\FrameworkBundle\Model\Localization\Localization $localization
      * @param \CommerceGuys\Intl\NumberFormat\NumberFormatRepositoryInterface $numberFormatRepository
      * @param \CommerceGuys\Intl\Currency\CurrencyRepositoryInterface $intlCurrencyRepository
-     * @param \Shopsys\FrameworkBundle\Component\CurrencyFormatter\CurrencyFormatterFactory $currencyFormatterFactory
      */
     public function __construct(
         CurrencyFacade $currencyFacade,
         Domain $domain,
         Localization $localization,
         NumberFormatRepositoryInterface $numberFormatRepository,
-        CurrencyRepositoryInterface $intlCurrencyRepository,
-        CurrencyFormatterFactory $currencyFormatterFactory
```

Comment:
The same thing as in `LocalizationListener` here https://github.com/shopsys/shopsys/pull/1425/files/f779df4570ae4178fd0eb68ceed548b624237aa9..cefda5143b308715f88518de21e2b8ed396b2fe5#r331780810

Reply:
thx, fixed in fa0438245 and 0470eb5c7